action-api/src/main/java/com/powsybl/action/AbstractLoadAction.java

```diff
@@ -18,7 +19,7 @@
  *
  * @author Anne Tilloy {@literal <anne.tilloy@rte-france.com>}
  */
-public abstract class AbstractLoadAction extends AbstractAction {
+public abstract class AbstractLoadAction extends AbstractAction implements ActionWithNetworkModification {
```

Comment:
@olperr1 I am not sure that the good solution is to add `ActionWithNetworkModification`. I was more thinking about a common method with a default implementation throw new Exception("Not implemented..."); What is your opinion?

Reply:
The purpose of this interface is more to be used in Rao and so to not have to implement toModification in the other Actions. But we can add a toModification which throws NotImplemented or I can try to implement a real toModification in the 7 other remaining actions even if it is not used (by Rao) for the moment. NB. I added the toModification in MultipleActionsAction because I thought it was necessary for Rao at one point but finally it is not necessary so it can be removed.

Reply:
I think the `MultipleActionsAction.toModification()` method can be kept, **without the `filter` clause**.
- If none of its actions throws an exception, the resulting `NetworkModificationList` will exactly match the behavior of the actions of the list;
- And if at least one of the actions doesn't implement the `toModification()` method, the `MultipleActionsAction.toModification()` will propagate the exception, which seems correct to me.

Reply:
This code was not used for Rao so it is removed